### PR TITLE
IGNORE [duplicate]

### PR DESCRIPTION
**Pull request by LukeErdy**

### CODE REVIEW

**Review by CadeTormanen — DISMISSED** [review]
LGTM!

**Review by amychisholm03 — CHANGES_REQUESTED** [review]
Please create a unit test in `server.test.js` that verifies that `getSimluationReport` works as intended.

File: test/unit/server.test.js
Comment (amychisholm03) [inline test/unit/server.test.js:1]: I'm not seeing any change?

**Review by amychisholm03 — APPROVED** [review]
CI is working (your test is passing) and merge conflicts are resolved. LGTM

**Review by CadeTormanen — APPROVED** [review]
Looks Goated

**Pull request merged**